core/common/src/test/java/com/alipay/sofa/rpc/common/json/JSONTest.java

```diff
@@ -38,4 +41,23 @@ public void test() {
         Assert.assertTrue(str.contains(JSON.CLASS_KEY));
     }
 
+    @Test
+    public void BeanWithMapOriginalObjectTest() {
```

Comment:
Pay a little attention to the method format.

Reply:
You're right. It looks like a class name right now... Maybe a name like testBeanWithMapSerialization() will be better, and I should look around methods in other test class before.

Reply:
Yes, it's better to modify it, there are no other problems about this PR. And we will submit another PR to fix the single test problem `testCustomThrowableDeserializerEnabled`.

Reply:
Glad to hear that. And it is strange that this test passed on my PC.

Reply:
This is usually caused when the unit tests run concurrently.